scripts/skills/perks/perk_rf_tempo.nut

```diff
@@ -95,10 +96,11 @@ this.perk_rf_tempo <- ::inherit("scripts/skills/skill", {
 
 	function onBeforeTargetHit( _skill, _targetEntity, _hitInfo )
 	{
-		if (this.m.APRecovery.len() != 0 && _skill.isAttack() && ::Tactical.TurnSequenceBar.isActiveEntity(this.getContainer().getActor()) && !_targetEntity.isTurnDone() && !_targetEntity.isTurnStarted())
+		if (this.m.AttacksRemaining > 0 && _skill.isAttack() && ::Tactical.TurnSequenceBar.isActiveEntity(this.getContainer().getActor()) && !_targetEntity.isTurnDone() && !_targetEntity.isTurnStarted())
```

Comment:
This perk has **Sword** as a requirement and you decided for this patch that all weapon tree perks are weapon locked, but I don't see anywhere in this file, where you enforce this restriction.